go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/looplab/fsm v0.3.0
 	github.com/metal-stack/go-ipam v1.8.5
 	github.com/metal-stack/masterdata-api v0.9.0
-	github.com/metal-stack/metal-lib v0.11.4
+	github.com/metal-stack/metal-lib v0.11.5-0.20230308090023-340574de5f20
```

Comment:
Can be pinned to latest release version.

Reply:
Pinned!